useful workflows/target_workflow_1/target_workflow_1.wdl

```diff
@@ -36,8 +36,7 @@ task foo {
     date '+%s' > digest.txt
     echo 'Input Digest:' >> digest.txt
 
-    # TODO: Remove 'echo' to use md5sum to generate digests:
-    echo 'TODO md5sum ~{input_file_1} ~{input_file_2}' >> digest.txt
+    md5sum ~{input_file_1} ~{input_file_2} >> digest.txt
```

Comment:
Just curious, why wasn’t this possible before?

Reply:
Before, `input_file_1` was just a random string, not a file that the command could access, so this would be equivalent to running, eg:
```
 % md5sum blahblahblah
md5sum: blahblahblah: No such file or directory
```